src/graphhopper/router.js

```diff
@@ -1,9 +1,50 @@
 import express from 'express';
 import graphHopperClient from './client.js';
 import logger from '../lib/logger.js';
+import * as gtfsRtClient from '../gtfs-rt/client.js';
+import { mergeAlertsIntoRoutes } from '../gtfs-rt/alerts.js';
 
 const router = express.Router();
-const routes = [
+router.use((req, res, next) => {
+  // you only want to cache for GET requests
+  if (req.method == 'GET') {
+    res.set('Cache-Control', 'public, max-age=120');
+  } else {
+    // for the other requests set strict no caching parameters
+    res.set('Cache-Control', `no-store`);
+  }
+  next();
+});
+
+router.get('/route-pt', async (req, res) => {
+  try {
+    const alertPromise = gtfsRtClient.getAlerts();
+    const graphHopperPromise = graphHopperClient.request({
+      method: 'get',
+      url: req.url,
+    });
+
+    const [alertResult, graphHopperResult] = await Promise.allSettled([
+      alertPromise, graphHopperPromise
+    ]);
+
+    if (graphHopperResult.status === 'rejected') throw graphHopperResult.reason;
+    if (alertResult.status === 'rejected') logger.error(alertResult.reason);
+
+    res.send(mergeAlertsIntoRoutes(alertResult.value, graphHopperResult.value.data));
```

Comment:
```suggestion
    res.json(mergeAlertsIntoRoutes(alertResult.value, graphHopperResult.value.data));
```

Reply:
Can you explain why you're asking for this change? What is wrong with the current way of doing it?

The use of `res.send()` is unchanged from your code on main. I am not familiar enough with this API to know what the difference is between calling `.send()` or `.json()`, and have chosen to make the minimum possible change to the code to accomplish my goal. If I change it, then I have to go research the API to make sure the way that you're preferring now will work and not break anything. That's significant added work and I'm not clear on what the benefit is.